src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,26 @@
-export const TodoInfo = () => {};
+import classNames from "classnames";
+import { UserInfo } from "../UserInfo";
+import { Todo } from "../../Types/Todo";
+import "./TodoInfo.scss";
+import usersFromServer from "../../api/users";
```

Comment:
Avoid using the `import` keyword in non-root components for data preparation. Instead, prepare the data in the App component and pass it down as props. This aligns with the guideline: 'Prepare data in one place (App component) and pass it to child components.'